sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_client_async.py

```diff
@@ -723,7 +725,7 @@ async def _client_ready_async(self):
         if not self._link:
             self._link = self._session.create_receiver_link(
                 source_address=self.source,
-                link_credit=self._link_credit,
+                link_credit=0,  # link_credit=0 on flow frame sent before client is ready
```

Comment:
calling out explicitly that the copy over is bringing the prefetch/link_credit = 0 fix to EH.

Reply:
moved all non-logging changes to separate pr: #34407